src/components/BlogPostOne/index.css

```diff
@@ -0,0 +1,39 @@
+.postone_container{
+    margin:auto;
+    max-width: 1300px;
+    padding-top: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 20px;
+}
+
+.PostImage{    
```

Comment:
It is good to have one naming style. Since you are using lowercase and separate words with an underscore, you could use it here as well.

Reply:
noted